src/containers/ItemList/index.js

```diff
@@ -11,43 +12,45 @@ class ItemList extends Component {
     }
   }
 
-
+  componentDidMount() {
+    this.props.loadItems()
+  }
 
   render() {
     return (
       <div className={this.props.category}>
-        {this.props.items.filter((element) => {
-          return element.category === this.props.category
-        }).map((element,idx) => {
+        {this.props.items.map((element,idx) => {
           return (
-            <Item
+            <Item 
```

Comment:
Change the item keys (and check in ItemList) to: 
{
id,
name,
price,
model,
dimensions,
created_at,
updated_at,
notes,
user_id,
category_id,
condition_id,
item_status_id,
image
}